### ISSUE TITLE
Docker build failure for ubuntu:bionic

### ISSUE THREAD

**Issue by regel** [spotify/dh-virtualenv#306]
Hello Jyrki, thanks for your work. Very helpful package 👍 

I used 1.1 without issues, and now I'm trying to upgrade to 1.2 and build the docker image.

The docker build instructions in README file works for debian stable, but not ubuntu:bionic. See build failure below. A suggestion of future improvement: publishing Docker images on docker hub would be really helpful, and this is something that can be automated from CircleCI using a free open source account. I can help with that.

Logs for the build:

```
docker build --build-arg distro=ubuntu:bionic   --tag dh-venv-builder .

...

Step 6/6 : RUN sed -i -re "1s/..unstable/~$(lsb_release -cs)) $(lsb_release -cs)/" debian/changelog     && dpkg-buildpackage -us -uc -b && mkdir -p /dpkg && cp -pl /dh-virtualenv[-_]* /dpkg     && dpkg-deb -I /dpkg/dh-virtualenv_*.deb
 ---> Running in 45a2df58223a
dpkg-buildpackage: info: source package dh-virtualenv
dpkg-buildpackage: info: source version 1.2-1~bionic
dpkg-buildpackage: info: source distribution bionic
dpkg-buildpackage: info: source changed by Jyrki Pulliainen <jyrki@dywypi.org>
 dpkg-source --before-build dpkg-build
dpkg-buildpackage: info: host architecture amd64
dpkg-checkbuilddeps: error: Unmet build dependencies: debhelper-compat (= 12)
dpkg-buildpackage: warning: build dependencies/conflicts unsatisfied; aborting
dpkg-buildpackage: warning: (Use -d flag to override.)
The command '/bin/sh -c sed -i -re "1s/..unstable/~$(lsb_release -cs)) $(lsb_release -cs)/" debian/changelog     && dpkg-buildpackage -us -uc -b && mkdir -p /dpkg && cp -pl /dh-virtualenv[-_]* /dpkg     && dpkg-deb -I /dpkg/dh-virtualenv_*.deb' returned a non-zero code: 3

```

The same error occurs on master branch, and 1.2 tag.

**Comment by regel** [comment]
Same error occurs with `--build-arg distro=debian:stretch` build argument

**Comment by regel** [comment]
With default arguments (`docker build   --tag dh-venv-builder .`) the build is OK for debian:buster

**Comment by jhermann** [comment]
Caused by https://github.com/spotify/dh-virtualenv/commit/8f51209a76a0fec6f8fd047d3d8348012b1d82c3

**Comment by jhermann** [comment]
Solved, brace for imcoming PR.

**Comment by jhermann** [comment]
Please test & close if working for you.

**Comment by jhermann** [comment]
Install test of doc-less Buster package on Jessie successful.

**Comment by nailor** [comment]
Should be fixed by #307, please re-open if that's not the case
